src/table/utils/__tests__/handle-accessiblity.spec.js

```diff
@@ -1,7 +1,11 @@
 import { expect } from 'chai';
 import * as handleAccessibility from '../handle-accessibility';
 
+<<<<<<< HEAD:src/table/cells/__tests__/handle-cell-focus.spec.js
+describe('handle-cell-focus', () => {
+=======
 describe('handle-accessibility', () => {
+>>>>>>> main:src/table/utils/__tests__/handle-accessiblity.spec.js
```

Comment:
glad to see that lint catch the issue :)

Reply:
hehe, well I should've caught it to begin with 🙃